app/controllers/static_controller.rb

```diff
@@ -0,0 +1,3 @@
+class StaticController < ApplicationController
+  def index; end
+end
```

Comment:
- [ ] Kindly as requested by the previous reviewer, make sure your app runs correctly on your machine.
- I have followed too the steps provided in the README and I'm still getting the same error. Please solve this
![rails_react](https://user-images.githubusercontent.com/79058364/178564014-2a2cb3bc-5073-49f2-97aa-4911f79dcc12.png)

Reply:
- [ ] Hello !!! After testing the project several times with the student's computer, I can confirm that the project runs perfectly without any errors, as you can see in this screenshot 👇🏻

![Screenshot taken on 2022-07-13 at 00-11-23 (1)](https://user-images.githubusercontent.com/79058364/178576721-9fca3913-9510-4147-abf1-ba98553ffb39.png)

- Please approve this!!